content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinioManualUpdate.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Updating MinIO from 1.6.58"
+description: "This article provides information on updating MinIO from 1.6.58 to newer versions."
+weight: 10
+tags:
+ - scaleminio
+ - scaledocker
+ - scaleapps
+---
+
+{{< toc >}}
+
+## Overview
+
+MinIO fails to deploy if you update your version 2022-10-24_1.6.58 Minio app to 2022-10-29_1.6.59 or later using the TrueNAS web UI.
+
+Your app logs display an error similar to the following:
+
+`ERROR Unable to use the drive /export: Drive /export: found backend type fs, expected xl or xl-single: Invalid arguments specified.`
+
+If you get this error after upgrading your MinIO app, use the app **Roll Back** function and return to 2022-10-24_1.6.58 to make your MinIO app functional again. 
+
+You will need [WSL2 (Windows Subsystem for Linux)](https://learn.microsoft.com/en-us/windows/wsl/install) if you are using a Windows computer.
+
+![MinioRollBack](/images/SCALE/MinioRollBack.png "MinIO Roll Back")
+
+## Manual Update Process
+
+### Create a New MinIO Deployment
+
+Follow the instructions [here]({{< relref "/content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/_index.md" >}}) to make a new, up-to-date MinIO deployment in TrueNAS. Make sure it is version 2022-10-29_1.6.59 or later.
+
+![MinIOClientISetupNewDeployment](/images/SCALE/MinIOClientISetupNewDeployment.png "Set Up New Deployment")
+
+### Download MinIO Client
+
+[Download the MinIO Client here](https://min.io/docs/minio/linux/reference/minio-mc.html?ref=docs) for your OS and follow the installation instructions. The MinIO Client (mc) lets you create and manage MinIO deployments via your system command prompt. 
```

Comment:
```suggestion
[Download the MinIO Client here](https://min.io/docs/minio/linux/reference/minio-mc.html?ref=docs) for your OS and follow the installation instructions. The MinIO Client (mc) lets you create and manage MinIO deployments via your system command prompt. 

Is there a reason to link as listed above versus minio.io/download?
```

Reply:
Yes! It's because Linux, Apple, and Windows all have different methods to install mc.exe.